--- /mnt/data/khal_baseline/WMS-sercoriego/sw.js	2026-09-10 22:14:23.632043344 +0000
+++ /mnt/data/khal_final_work/WMS-sercoriego/sw.js	2026-09-10 22:13:55.262595567 +0000
@@ -1,4 +1,4 @@
-const APP_VERSION = '2026.09.10-ayuda-empaque-visible-v159';
+const APP_VERSION = '2026.09.10-zebra-raw-bridge-v160';
 const CACHE = `sercoriego-lite-wms-${APP_VERSION}`;
 const CACHE_PREFIX = 'sercoriego-lite-wms-';
 
